### PR TITLE
add why use this section

### PR DESCRIPTION
**Pull request by lukewestby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,26 @@ store
...
+
+Having used and followed the progression of Redux and the Elm Architecture, and
+after trying other effect patterns for Redux, we came to the following
+conclusion:
+
+> Synchronous state transitions caused by returning a new state from the reducer
+> in response to an action are just one of all possible effects an action can
+> have on application state.
+
+Many of the other possible methods for handling effects in Redux which focus on
```
Comment (dustyburwell) [inline README.md]: This sentence runs on with what appear to be a few typos.

File: README.md [hunk 2]
```diff
@@ -76,6 +76,26 @@ store
...
+action-creators guide the user either explicitly or implicitly to understand
+asynchronous effects as a significantly different concept than reducer state
+transitions simply because they are asynchronous. With `redux-loop`, the reducer
+doesn't just decides what happens _*now*_ due to a particular action, it decides
```
Comment (dustyburwell) [inline README.md]: s/decides/decide

File: README.md [hunk 3]
```diff
@@ -76,6 +76,26 @@ store
...
+what happens _*next*_. All of the behavior of your application can be traced
+through one place, and that behavior can be easily broken apart and composed
+back together. This is one of the most powerful features of the Elm
+Architecture, and with `redux-loop` it can be a feature of Redux as well.
```
Comment (dustyburwell) [inline README.md]: s/Architecture/architecture. I'd also replace "can be" with "is". Just a bit more affirmative, that way.

File: README.md (hunk 1 above)
Comment (lukewestby) [inline README.md]: I should really have paid more attention in English class

**Comment by lukewestby** [comment]
@dustyburwell look good?

File: README.md [hunk 4]
```diff
@@ -76,6 +76,28 @@ store
...
+with action-creators, incorrectly teach the user that asynchronous effects are
+fundamentally different from synchronous state transitions. This separation
+encourages divergent and increasingly specific means of processing particular
+types effects. Instead, we should focus on making our reducers powerful enough
+to handle asynchronous effects as well as synchronous state transitions. With
+`redux-loop`, the reducer doesn't just decide what happens _*now*_ due to a
+particular action, it decides what happens _*next*_. All of the behavior of your
+application can be traced through one place, and that behavior can be easily broken
+apart and composed back together. This is one of the most powerful features of the
+Elm architecture, and with `redux-loop` it is a feature of Redux as well.
```
Comment (dustyburwell) [inline README.md]: Perhaps a link to the Elm project page to demystify what that is for folks unfamiliar?

**Comment by dustyburwell** [comment]
Otherwise :+1:

**Pull request merged**